examples/using-multiple-themes/README.md

```diff
@@ -0,0 +1,60 @@
+<p align="center">
+  <a href="https://www.gatsbyjs.org">
+    <img alt="Gatsby logomark" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
```

Comment:
should the logomark not better in the example repo?

Reply:
I'm going to leave this for Laurie or Obinna.